ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+**DO NOT ask questions using GitHub issues (only bug or feature requests please see http://loopback.io/doc/en/contrib/Reporting-issues.html#asking-questions)**
+**Please ask questions at https://groups.google.com/forum/#!forum/loopbackjs or https://gitter.im/strongloop/loopback**
+**Immediate support is available through our subscription plans, see https://strongloop.com/node-js/subscription-plans**
+
+### Bug or feature request
+
+- [ ] Bug
+- [ ] Feature request
+
+### Description
+
+
+
+### Expected result
+
+
+### Actual result (if bug)
+
+
+
+### Link to sample repo to reproduce issue (if bug)
```

Comment:
Let's move this more to the top, before "expected" result.

In my experience, a good bug report has the following form:

 - steps to reproduce
 - actual result
 - expected result
 - additional information like Node/LoopBack versions, etc.